src/pages/repositories.html

```diff
@@ -1,196 +1,181 @@
+                        </div>
+                        <div ng-if="location.local" class="actions-bar-inline pull-right">
+                            <button
+                                tooltip="{{'repos.edit.common.settings' | translate}}"
+                                tooltip-placement="top" tooltip-trigger="mouseenter"
+                                class="btn btn-link px-0"
+                                ng-click="openActiveLocationSettings()">
+                                <span class="icon-settings"></span>
+                            </button>
+                            <a class="btn btn-link px-0 pull-right" href="license"
+                               tooltip="{{'repos.view.update.licenses' | translate}}"
+                               tooltip-placement="top" tooltip-trigger="mouseenter">
+                                <span class="icon-key"></span>
+                            </a>
+                        </div>
+                        <div ng-if="isAdmin() && !location.local" class="actions-bar-inline pull-right">
+                            <button class="btn btn-link p-0"
+                                    tooltip="Edit location" tooltip-placement="top" tooltip-trigger="mouseenter"
+                                    ng-click="editLocation(location)">
+                                <em class="icon-edit"></em>
+                            </button>
+                            <button class="btn btn-link p-0 secondary"
+                                    tooltip="{{'detach.location' | translate}}" tooltip-placement="top" tooltip-trigger="mouseenter"
+                                    ng-click="deleteLocation(location.uri)">
+                                <em class="icon-close"></em>
+                            </button>
+                        </div>
+                    </div>
+                    <div ng-if="hasActiveLocation()" class="pt-2 mb-3">
+                        <div class="alert alert-warning no-repositories-alert"
+                             ng-if="!getRepositoriesFromLocation(location.uri).length && !location.errorMsg">
+                            {{'no.repos.in.current.location' | translate}}
+                        </div>
+                        <div class="alert alert-danger" ng-if="location.errorMsg">
+                            {{location.errorMsg}}{{'repos.refresh.page.msg' | translate}}
+                        </div>
+                        <div ng-if="getRepositoriesFromLocation(location.uri).length" class="mb-1">
+                            <table id="wb-repositories-repositoryInGetRepositories"
+                                   class="table table-hover table-striped" aria-describedby="Repositories table">
+                                <tbody>
+                                <tr class="repository"
+                                    ng-repeat="repository in getRepositoriesFromLocation(location.uri) | orderBy: ['type === \'system\'', 'id', 'location']"
```

Comment:
Order by location is not needed here as the repositories are all from one location.